lib/themes/seekAsia/atoms/private/palette.js

```diff
@@ -0,0 +1,11 @@
+export const white = '#fff';
+export const black = '#000';
+export const blue2 = '#298EB9';
+export const blue3 = '#94C9E0';
+export const blue5 = '#EEF8FC';
+export const alert = '#ff4b4b';
+export const alertBackground = 'rgba(255,75,75, 0.15)'; // transparent seekAsia alert color
+export const completion = '#63b209';
+export const grey1 = '#333';
+export const grey4 = '#ccc';
+export const candidate = '#0c4b85'; // SEEK Asia name for this color
```

Comment:
Nice one 👏

I've been meaning to do this for the other themes too. Great to see the pattern introduced.